src/client/table_client.rs

```diff
@@ -296,7 +301,9 @@ impl ObTableClientInner {
                  table_name:{}, err:{}",
                 table_name, error
             );
-            self.get_or_refresh_table_entry_non_blocking(table_name, true)?;
+            let _ = self.refresh_sender.try_send(table_name.to_owned());
+            warn!("ObTableClientInner::on_table_op_failure: try to refresh schema actively, table_name:{table_name}");
```

Comment:
Just so you know, it is necessary here since we have two scenarios to send a signal to refresh threads.
1. "Refreshing Error Code"
2. Too many times